requirements.txt

```diff
@@ -10,7 +10,7 @@ rich==13.4.1
 toml==0.10.2
 translators==5.9.1
 pyttsx3==2.90
-Pillow==10.3.0
+Pillow==9.5.0
```

Comment:
Why is Pillow downgraded?

Reply:
FreeTypeFont object has no attribute getsize error

Reply:
Will check